src/styles/pages/checkout/checkout.scss

```diff
@@ -94,6 +94,42 @@
         padding-left: $space-default * 2;
       }
     }
+
+    &.infobox-color-widget {
+      padding-bottom: 15px;
+      font-size: 14px;
+      color: $text-color-inverse;
+      text-transform: uppercase;
+      background: $CORPORATE-PRIMARY;
+      h3 {
+        font-size: 1.09375rem;
+        color: $text-color-inverse;
+      }
+      a {
+        color: $text-color-inverse;
+        text-decoration: underline;
+        text-transform: none;
+      }
+      .infobox-count {
+        font-size: 36px;
+        line-height: 1;
+      }
+      .infobox-count-large {
+        font-size: 60px;
```

Comment:
$font-size-lg * 3 or in rem

Reply:
according to Inken it should be 60px so I'm doing 3.75rem